### PR TITLE
Make issue templates more adequate to G-Node/gogs

### PR DESCRIPTION
**Pull request by yarikoptic**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -2,4 +2,4 @@ blank_issues_enabled: false
 contact_links:
   - name: Gogs Community Forum
     url: https://discuss.gogs.io/
-    about: Please ask and answer questions here.
+    about: Please ask and answer questions about Gogs (not g-node's fork) here.
```
Comment (achilleas-k) [inline .github/ISSUE_TEMPLATE/config.yml]:
```suggestion
    about: Please ask and answer questions about Gogs (not the G-Node's fork) here.
```

File: .github/ISSUE_TEMPLATE/security.md [hunk 2]
```diff
@@ -7,6 +7,6 @@ about: Report security vulnerability for this project
 <!--
 
 Please create a dummy issue with high-level description of the security vulnerability,
-then report details to u@gogs.io privately, thank you!
+then report details to Achilleas Koutsou <achilleas.k@gmail.com> privately, thank you!
```
Comment (achilleas-k) [inline .github/ISSUE_TEMPLATE/security.md]:
```suggestion
then report details to Achilleas Koutsou <koutsou@bio.lmu.com> privately, thank you!
```
Use my work email instead.

**Comment by achilleas-k** [comment]
Thanks.  I made a couple of changes, if you don't mind.

**Comment by yarikoptic** [comment]
accepted suggestions ;)

**Comment by achilleas-k** [comment]
Thanks.  Will merge as soon as the checks finish.

**Pull request merged**